ci.sh

```diff
@@ -41,7 +41,7 @@ python -m pip install -U pip uv -c test-requirements.txt
 python -m pip --version
 python -m uv --version
 
-python -m uv pip install build
+python -m uv pip install build -c test-requirements.txt
```

Comment:
This could also use an env var:
```suggestion
PIP_CONSTRAINT=test-requirements.txt python -m uv pip install build
```